src/data_gradients/managers/abstract_manager.py

```diff
@@ -257,3 +259,34 @@ def _create_samples_iterated_warning(self) -> str:
         msg_train = f"Train set: {self._train_iters_done} out of {total_train_samples} samples were analyzed{portion_train}.\n"
         msg_val = f"Validation set: {self._val_iters_done} out of {total_val_samples} samples were analyzed{portion_val}.\n "
         return msg_head + msg_train + msg_val
+
+    @staticmethod
+    def _get_grouped_feature_extractors(default_config_name: str, config_path: str, feature_extractors) -> Dict[str, List[AbstractFeatureExtractor]]:
```

Comment:
`feature_extractors` missing typing.
Because the typing is huge, maybe we can save it as a constant somewhere and re-use it whenever needed, this will still work for TypeChecking